### ISSUE TITLE
Summon Night (PSX)

### ISSUE THREAD

**Issue by someblackmage** [IcySon55/Kuriimu#437]
Issue: plugin request
Console: Playstation
Format extension(s): .bin
Type: archive
Game Name: Summon Night

First 8 bytes of the file(s):
.bin - 00 FF FF FF FF FF FF FF - [41]

More details:
I was told that the format used for this game was not supported by any plugins. I hope someone will remedy that.

Sample File:
https://www.dropbox.com/s/pu6kjx0xhl9wi30/Summon%20Night.zip?dl=0

**Comment by someblackmage** [comment]
The bytes here are somewhat off. You might want to search for a ROM on the internet to find if this is okay.

**Comment by IcySon55** [comment]
You didn't provide a sample file... (keep in mind not to attach it to github)

**Comment by someblackmage** [comment]
Sorry for the late reply. But what kind of file are you looking for?
Because I have the .bin.

**Comment by someblackmage** [comment]
How do I provide one anyway?

**Comment by piratesephiroth** [comment]
what is even this .bin you speak of?

**Comment by someblackmage** [comment]
The .bin is the editable file I believe. I can attach it to an email if you
wish.

**Comment by piratesephiroth** [comment]
you can upload to one of many file hosting services, like https://comfy.moe/

**Comment by someblackmage** [comment]
Alright. BTW who are you?

**Comment by someblackmage** [comment]
Also, I downloaded the files off a site. Should I link to that?

**Comment by piratesephiroth** [comment]
I am batman.

And I ask again: WHAT are these files you talk about? disk images?

**Comment by IcySon55** [comment]
"I was told that the format used for this game", Correct, formats are supported one at a time and this game has not crossed our radars until now. But with samples of the files in question, what do you expect us to do?

"the files you are supposed to edit", Yes these, if you have some of them, then upload them to some online storage and link them.

**Comment by someblackmage** [comment]
The thing is I downloaded them from a website and, surprisingly, it came
that way and I don't know if I should link to the site or if I should
upload them myself. What should I do?

**Comment by someblackmage** [comment]
I added the sample file

**Comment by piratesephiroth** [comment]
as I suspected, that's the disc image file

**Comment by someblackmage** [comment]
oh....... I'm a dummy dummy then. :|

**Comment by onepiecefreak3** [comment]
TLDR: Format proved to be a disc image all along, the issue gets closed.

Since this issue was for .bin/.cue disc images, without the issuer knowing that those are standard disc images, it gets closed.. Kuriimu2 has no incentive to support them (yet), since many tools exist to read and edit them in some way.
